### PR TITLE
Release

### PR DESCRIPTION
**Pull request by Teslothorcha**

### CODE REVIEW

**Comment by Teslothorcha** [comment]
accepted

File: console.py [hunk 1]
```diff
@@ -83,25 +91,24 @@ def do_destroy(self, line):
             storage.save()
 
     def do_all(self, args):
-        all_obj = storage.all()
+        """Type method all"""
+        all_obj = [str(v) for v in storage.all().values()]
```
Comment (edward0rtiz) [inline console.py:95]: Improved line using list comprehension

**Review by edward0rtiz — APPROVED** [review]
Improved code of def all. tested at console. It works for console 1.0 a for def Update. work in progress for tasks 15 and 16

File: console.py [hunk 2]
```diff
@@ -83,25 +91,24 @@ def do_destroy(self, line):
...
         if not args:
-            for i in all_obj.values():
```
Comment (Teslothorcha) [inline console.py:88]: change printing formart

File: console.py [hunk 3]
```diff
@@ -83,25 +91,24 @@ def do_destroy(self, line):
...
-                print('{}'.format(i))
-        else:
+            print('{}'.format(all_obj))
```
Comment (Teslothorcha) [inline console.py:97]: changed format to list

**Review by Teslothorcha — COMMENTED** [review]
aprooved , missing task 15 and 16

File: console.py (hunk 2 above)
Comment (edward0rtiz) [reply console.py:88]: Got it

**Pull request merged**